app/Http/Controllers/MemberController.php

```diff
@@ -12,10 +12,29 @@ class MemberController extends Controller
      *
```

Comment:
Please add the `.phpintel` folder to your `.gitignore` file. You don't need these files in your repository.

Reply:
need update git for it, i adrealy at this line to .gitignore